### DISCUSSION TITLE
Flet - build Pyodide apps with Flutter UI

### DISCUSSION THREAD

**Discussion by FeodorFitsner** [pyodide/pyodide#3564] [Show and tell]
Hi everyone!

My name is Feodor and I'm the founder of Flet framework (https://flet.dev) that allows building multi-platform Python app with Flutter UI. On behalf of Flet community I'd like to thank you for Pyodide! :) Pyodide made possible running Flet apps entirely in the browser! I saw Pyodide users are asking about Kivy, or PySide or even tkinter support in Pyodide which could be quite a challenge to implement. There is also pyscript project mixing Python with HTML.

Flet allows adding a stunning Flutter-based UI to your Pyodide apps! I did a blog post with a simple getting started and a small gallery of pre-published apps: https://flet.dev/blog/standalone-flet-web-apps-with-pyodide

Live demos:
* [To-Do](https://flet-todo.pages.dev/) ([source](https://github.com/flet-dev/examples/blob/main/python/apps/todo/todo.py))
* [Calculator](https://flet-calc.pages.dev/) ([source](https://github.com/flet-dev/examples/blob/main/python/tutorials/calc/calc.py))
* [Icons browser](https://flet-icons-browser.pages.dev/) ([source](https://github.com/flet-dev/examples/blob/main/python/apps/icons-browser/main.py))
* [Solitaire](https://flet-solitaire.pages.dev/) ([source](https://github.com/flet-dev/examples/tree/main/python/tutorials/solitaire/solitaire-final-part1))

Let me know what you think!

**Comment by rth** [comment]
Thanks for sharing! Looks very nice and really smooth interactive experience as well!

Don't hesitate to add Flet to the [related projects](https://pyodide.org/en/stable/project/related-projects.html) page (and add a new section if necessary).

**Comment by gabrielfougeron** [comment]
Wow, looks great! Thanks for sharing :-)
